src/ci_tools.ts

```diff
@@ -53,6 +56,18 @@ async function run(originalArgv: string[]): Promise<void> {
     printHelp();
     process.exit(1);
   }
+
+  const mode: string = argv.mode || DEFAULT_MODE;
+
+  if (isBoolean(mode) || (mode !== 'dotnet' && mode !== DEFAULT_MODE)) {
+    console.error('Mode must be set to `dotnet` or `node`. \nDefault is `node`');
+    process.exit(1);
+  }
+
+  if (argv.mode === undefined) {
```

Comment:
```suggestion
  if (argv.mode == null) {
```

Ich bin mir wirklich nicht sicher, wie oft ich das jetzt erwähnt habe. Wenn du wenigstens `== undefined` für `null`/`undefined` nutzen würdest.

So habe ich den Eindruck, dass du noch nicht verstanden hast, wo der Unterschied liegt und warum in diesem *einen Fall*  `==` lebensnah betrachtet "sicherer" ist.

Reply:
Ist lediglich eine Angewohnheit. Wenn man das zwei Jahre lang so macht dann denkt man da nicht immer direkt dran